lab_2_tokenize_by_bpe/main.py

```diff
@@ -58,6 +105,30 @@ def train(
     :param num_merges: required number of new tokens
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(num_merges, int):
+        return None
+    dict_of_token_pairs = count_tokens_pairs(word_frequencies)
+    if dict_of_token_pairs is None:
+        return None
+    list_of_max_token_pairs = []
+    for i in range(0, num_merges):
+        if len(list_of_max_token_pairs) == num_merges:
+            break
```

Comment:
Is it possible to get here given that you merge tokens `num merges` times? (line 114)

Reply:
I realized my mistake, I will try to use while